features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/members/RoomMemberListView.kt

```diff
@@ -71,6 +71,7 @@ import io.element.android.libraries.matrix.ui.components.MatrixUserRow
 import kotlinx.collections.immutable.ImmutableList
 import io.element.android.libraries.ui.strings.R as StringR
 
+@OptIn(ExperimentalMaterial3Api::class)
```

Comment:
According to the linter, there is no experimental API

Reply:
Indeed, my AS was out of sync.